provider/pkg/logging/time_ordered_log_set.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016-2019, Pulumi Corporation.
+// Copyright 2016-2022, Pulumi Corporation.
```

Comment:
Should this file be in cloud-ready-checks instead?

Reply:
I think this logic is more specific to provider logging, so I decided to leave it here. The await library returns the messages and it's up to the client how to handle them.